src/Course/StateT.hs

```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE FlexibleContexts #-}
```

Comment:
Why flexible contexts?

Reply:
just stuff that was put by emacs when I was playing with something